### PR TITLE
Fix denials that cause systemd to crash

### PR DESCRIPTION
**Pull request by Thor-x86**
The pull request fixes issue #282

### CODE REVIEW

File: container.te [hunk 1]
```diff
@@ -194,6 +194,21 @@ allow container_runtime_domain self:netlink_kobject_uevent_socket create_socket_
...
+	gen_require(`
+		type init_t;
+		type shadow_t;
+		type systemd_logind_t;
+		class process siginh;
+		class capability net_admin;
+	')
+
+	allow init_t container_user_t:process siginh;
+	allow init_t shadow_t:file { open read };
```
Comment (rhatdan) [inline container.te:208]: Don't add this or the next line stick to container_user_t. and see if that works.

Comment (Thor-x86) [reply container.te:208]: It doesn't work but AVC shows nothing even though I ran `sudo semodule -DB`

Comment (rhatdan) [reply container.te:208]: If setenforce 0, and it still fails, then it is not SELinux blocking access.

What podman command are you running?

**Comment by rhatdan** [comment]
Since I have not heard back, closing, reopen if it happens again.

**Pull request closed without merging**